src/Tizen.TV.UIControls.Forms/Renderer/UIControlsExtensions.cs

```diff
@@ -53,6 +52,9 @@ public static MauiAppBuilder UseTizenTVUIControls(this MauiAppBuilder builder)
 
                     //TODO
                     //handlers.AddCompatibilityRenderer(typeof(AnimatedNavigationPage), typeof(AnimatedNavigationPageRenderer));
+
+                    // This is a workaround for wrong ListView behavior
+                    handlers.AddHandler<ListView, TVListViewRenderer>();
```

Comment:
I think that it seems better to move this custom renderer into sample project.🤔 Because this render is for fix ListView issue until MAUI is fixed. it is not TV UI control.

Reply:
Ok I will fix it.